src/components/SubLanguageSelector.tsx

```
@@ -23,7 +23,7 @@ const SubLanguageSelector = ({
 }: SubLanguageSelectorProps) => {
   const navigate = useNavigate();
 
-  const { language, subLanguage, setSearchText } = useAppContext();
+  const { selectedLanguage, selectedCategory, setSearchText } = useAppContext();
```

ESLint

[error] 'selectedLanguage' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)